app/views/record/_extended_info.html.erb

```diff
@@ -6,9 +6,11 @@
 
 <h3 class="section-title">More information</h3>
 <ul class="list-moreinfo">
-  <% if @record.eds_publication_type.present? %>
+  <%# Use get_item_data({name:'TypePub'}), not eds_publication_info, to match
```

Comment:
I'm not sure this comment helps as the info it talks about is actually in the helper... I recommend just not documenting this as the helper is clear enough as to what it does.